e2e/tests/integrations/view-integrations-in-overview.test.ts

```diff
@@ -3,5 +3,4 @@ import { expect, test } from "@playwright/test";
 test("view my integrations in overview", async ({ page }) => {
   await page.goto("/nl/integraties");
   await expect(page.locator("h2").getByText("Mijn integraties")).toBeVisible();
-  await expect(page.getByText("Geen integraties gevonden")).toBeVisible();
```

Comment:
Why has this been removed? `view my integrations in overview` still has to be implemented

Reply:
It is because with the new tests running in parallel.
The integrations are not "empty" since there's a test for creating one...

Reply:
Ok, maybe we will have to change this behaviour later to be sequential.